### DISCUSSION TITLE
Welcome to vnet-backend Discussions!

### DISCUSSION THREAD

**Discussion by muremwa** [wild-times/vnet-backend#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by muremwa** [comment]
@SamuelNw @georgeshei Questions?

**Comment by muremwa** [comment]
You can add the secret key by adding a secret key to `vnet/secret_key.txt`.
This key can be gotten from:
```python
from django.core.management.utils import get_random_secret_key

secret_key = get_random_secret_key()
```
Copy the key generated to `vnet/secret_key.txt`.

**Comment by muremwa** [comment]
For the Azure Communication Services (ACS) connection string, add it to `vnet/acs.txt` once you receive it.

**Comment by SamuelNw** [comment]
Yes all done. And application is up and running. But while signing up, just after clicking on the sign up option, I get this error of :
"
ClientAuthenticationError at /

(Denied) Request was out of the allowed time difference.
Code: Denied
Message: Request was out of the allowed time difference....." 

Is this a problem with my development setup or the application?

**Reply by SamuelNw** [reply]
I see several options on google (talking of my local machine and BIOS and so on) but I would like to know if someone else is experiencing this, @georgeshei ?

**Reply by SamuelNw** [reply]
In fact, the above error now can't even let me view the home page. It comes up as soon as the link opens up on the browser.

**Reply by muremwa** [reply]
Is the error still there?

**Reply by SamuelNw** [reply]
You asking because of the .gitignore files you changed?

**Reply by muremwa** [reply]
No, unrelated

**Comment by SamuelNw** [comment]
@georgeshei when installing the dependencies from requirements.txt, comment out "dataclasses==0.8". For some reason it was not installing and stopping the whole process, meaning other packages were not getting installed.

**Reply by muremwa** [reply]
`dataclasses` is found in the standard library, if you're using python 3.7+
It won't be a problem if so.

**Comment by SamuelNw** [comment]
@georgeshei @muremwa I also think the buttons on the home page on sign up, sign in and get started should have the anchor tags wrapping the "button" tag instead of the vice versa as it is right now. Not unless @muremwa you did update this after I cloned the repo.

**Reply by muremwa** [reply]
@SamuelNw you can make those changes and make a pull request.

**Reply by SamuelNw** [reply]
I shall

**Reply by SamuelNw** [reply]
Created the pull request at https://github.com/wild-times/vnet-backend/pull/9. anyone wanna merge?

**Comment by muremwa** [comment]
@SamuelNw @georgeshei Should we make the repositories public?

**Comment by SamuelNw** [comment]
Not yet

**Reply by muremwa** [reply]
Can I see the error message? as a screenshot.

**Reply by SamuelNw** [reply]
![image](https://user-images.githubusercontent.com/64871293/187593638-df3b3779-5273-4f21-a381-85b4a7c3f354.png)

**Reply by SamuelNw** [reply]
Problem solved. The time on my machine being a bit behind was causing the error. now we good.

**Reply by muremwa** [reply]
👍🏾

**Comment by SamuelNw** [comment]
There is also a slight typo on:
![image](https://user-images.githubusercontent.com/64871293/187595420-36f6cc9c-1ed0-4721-b7ad-005edbdd9076.png)
I think what you meant on number 8, second sentence, is Then generates "answer"...

**Reply by muremwa** [reply]
Yeah yeah I need to fix it

**Reply by muremwa** [reply]
[334f334db0410b1741fa19a07ae072ac44355b91 ](https://github.com/wild-times/vnet-frontend/commit/334f334db0410b1741fa19a07ae072ac44355b91) fixes
